modules/op-lightspeed-utility-commands.adoc

```diff
@@ -0,0 +1,35 @@
+// This module is included in the following assemblies:
+// * tkn_cli/op-tkn-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-lightspeed-utility-commands_{context}"]
+= {ols} utility commands
+
```

Comment:
Need to add opc needs lightspeed configuration to work. add the notes for the same